AffiliatedPackages.md

```diff
@@ -0,0 +1,111 @@
+	community standards to foster their existance as an open
+	source community.
+2. They gain the ability to easily reach out to the Planetary
+	Software TC for consultation or mentorship on technical
+	or governance issues.
+3. Being recognized as a PlanetaryPy affiliated package lets others
+	know that their project conforms to community best practices
+	which are likely to result in long-term success for their
+	project.
+
+## Becoming an Affiliated Package
+
+If you are a developer of a planetary science package and would like your
+package to become affiliated with the PlanetaryPy Project, please take
+a look at the [instructions for proposing an affiliated package][review-process]. We
+recommend that you also take a look at the [guidelines for reviewing
+affiliated packages][guidelines] since this will give you a sense of whether
+your package is ready for review. Broadly speaking, your package
+should:
+
+* Be potentially useful to planetary scientists. This can mean
+  useful to a specific sub-domain of planetary science, or more broadly useful
+  to a large fraction of planetary science (or beyond, as long as it is also
+  useful for planetary science).
+
+* Be written in a way that is readable and understandable by others. While 
+  not a strict requirement, we also provide coding guidelines that will make 
+  your code easier to read by members of the community.
+
+* Use classes and functions from the `planetarypy` core package wherever possible
+  and appropriate, and (as much as possible) avoid duplication with other packages
+  in the PlanetaryPy ecosystem. This facilitates re-use of code and sharing of resources.
+
+* Have documentation that adequately explains the use of the package. Additionally,
+  user-facing classes and functions should all have docstrings. We suggest using 
+  sphinx for your documentation.
+
+* Make a best-effort to include an easy-to-run test suite that covers its intended
+  functionality. We realize this is not always possible, but when it is, a test 
+  suite is a crucial element of stable software and reproducible science.
+
+* Be compatible with Python 3.6 and above.
+
+* Be open to contributions from others. While this most straightforwardly means it 
+  follows a Github-based open development model (like the PlanetaryPy core package),
+  alternative approaches are perfectly valid as long as they are consistent with
+  basic principles of open source. (E.g., an OSI-approved license)
+
+In addition, you should make an effort to connect with the PlanetaryPy
+developer community, including developers from the core planetarypy
+package or any related affiliated packages. If your package is
+determined to meet the above standards, it will be accepted and
+added to the affiliated package registry. Note however that if
+packages become unmaintained or do not meet the standards anymore,
+they may be removed from the list of affiliated packages.
+
```

Comment:
does this need a timeframe where this would happen? i guess it's more an issue for the TC to even notice that something isn't maintained. But how about a "(1)tests are failing AND (2) no commit in one year" trigger for removal action?

Reply:
I think I'd like to not specify that here, lest we tie the TC's hands too early or too tightly.  Right now, it is worded broadly, and gives the TC a lot of latitude.  If we think we need to be exacting about the conditions, I guess we could do that, but I'd like to refrain until we need to.